test/test_getting_started.rb

```diff
@@ -0,0 +1,37 @@
+require "test/unit"
+
+class TestGettingStarted < Test::Unit::TestCase
+  def setup
+    puts "installing rails"
+    a = `gem install rails --no-document`
+    puts "which: #{`which rails`}"
+    puts "rails version: #{`rails --version`}"
+  end
+
+  def test_build_succeeds
+    assert(system("rm -rf tmp; mkdir -p tmp"), "tmp dir is replaced")
+
+    Dir.chdir("tmp") {
+      cmd = "yes | YES_ALL=1 rails new Foobar -T --skip-coffee --webpack --database=postgresql -m '../rails-kickoff-template.rb'"
```

Comment:
using `yes` is generally fine, but for this question on line 413 `yes?("Is this template being run on an existing application? (usually no)")` The typical use of the application is to say no here. I agree it's a confusingly worded question and happy to get suggestions for better wording that might result in a yes default.

There are two ways to use this template: 1) to create a new application, 2) to apply to an existing application:

https://guides.rubyonrails.org/rails_application_templates.html

> You can use the app:template Rake task to apply templates to an existing Rails application.

This lets us apply new defaults to existing applications. It obviously can't be done blindly (every change will need to be reviewed), but my hope is it'll let us keep our default approach on applications a little more standard even as we update defaults.